### PR TITLE
feat(bix): prevent running control server inadvertantly

### PR DESCRIPTION
**Pull request by JTarasovic**
I think we want a separate command for using control server against a remote cluster. Trying to get it working so that one could specify a slug was getting a bit messy and convoluted. The steps are largely the same but the postgres connection details would need to be figured out and passed. I think we can land this to keep us safe and add the remote cluster stuff later.

### CODE REVIEW

File: platform_umbrella/apps/kube_services/lib/kube_services.ex [hunk 1]
```diff
@@ -2,16 +2,4 @@ defmodule KubeServices do
...
-
-  def get_current_events do
-    conn = CommonCore.ConnectionPool.get!()
-
-    operation = K8s.Client.list("v1", "Event")
-
-    with {:ok, result} <- K8s.Client.run(conn, operation) do
-      result
-    end
-  end
```
Comment (JTarasovic) [inline platform_umbrella/apps/kube_services/lib/kube_services.ex:14]: Not used anywhere in our codebase.

File: platform_umbrella/apps/kube_services/lib/kube_services.ex [hunk 2]
```diff
@@ -2,16 +2,4 @@ defmodule KubeServices do
...
-
-  def cluster_type, do: Application.get_env(:kube_services, :cluster_type, :dev)
```
Comment (JTarasovic) [inline platform_umbrella/apps/kube_services/lib/kube_services.ex:16]: Was only used in `KubeServices.Connection` below.

File: platform_umbrella/apps/kube_services/lib/kube_services/connection.ex [hunk 3]
```diff
@@ -1,12 +0,0 @@
-defmodule KubeServices.Connection do
```
Comment (JTarasovic) [inline platform_umbrella/apps/kube_services/lib/kube_services/connection.ex:1]: Not used anywhere.

**Review by elliottneilclark — APPROVED** [review]

**Comment by elliottneilclark** [comment]
Thank you!

**Pull request merged**